plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/compaction/oned/OneDimensionalCompactor.java

```diff
@@ -9,10 +9,6 @@
  *******************************************************************************/
 package org.eclipse.elk.alg.layered.compaction.oned;
 
-// elkjs-exclude-start
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.PrintWriter;
 // elkjs-exclude-end
```

Comment:
Could you also remove the end command?

Reply:
Done.
I have also checked if there are more now unmatched pairs, but all looks good now.